src/layout/generic-input/utils/generic-input-utils.js

```diff
@@ -261,3 +260,21 @@ export function getNewQuestionPlaceholder(components, activeComponent) {
     weight,
   };
 }
+
+/**
+ * Get new loop placeholder
+ *
+ * It finds from a list of components if it have at least one sequence
+ *
+ * @param  {object} components        List of components
+ * @return {string} true if component have a sequence
+ */
+export function getNewLoopPlaceholder(components) {
+  let componentHaveSequence = true;
```

Comment:
I think all this method can be replaced with something like : 
`return Object.values(components).find(component => component.type === SEQUENCE);`